web/src/settings_emoji.js

```diff
@@ -126,6 +126,13 @@ export function populate_emoji() {
             }
             return "";
         },
+        callback_after_render() {
+            // While rendering a list, the ListWidget at the end of the provided list
+            // checks if it needs to render a message in case the container is empty.
+            // However, the provided emoji list might contain deactivated emojis that will
+            // not be displayed. Therefore, an additional check is required here.
+            ListWidget.render_empty_list_message_if_needed($emoji_table);
```

Comment:
> Posted one comment. And I cannot see the empty message in "Custom emoji" panel if there are no custom emojis for the realm.

@sahil839 fixed it here. You can take a look. Thanks!

Reply:
Can a better solution for this would be just exclude deactivated emojis from the list at the start only, i.e. before passing the data to `ListWidget`.